recipe/run_test.py

```diff
@@ -28,8 +28,10 @@ def run_test():
         print("WARNING: Skipping runtime tests on aarch64 as GLIBC version is lower than 2.27")
         return nvvm.get_version() is not None
 
-    if not test():
-        return False
+    # Skip this test as it looks for the `libcuda.so` driver library,
+    # which is not included in the Docker image used here.
+    #if not test():
+    #    return False
```

Comment:
The Docker image used here doesn't have the `libcuda.so` driver library, which [Numba tests for]( https://github.com/numba/numba/blob/249c8ff3206928b486346443ec148508f8c25f8e/numba/cuda/cudadrv/libs.py#L88-L99 ). As a result, this test fails. So here we skip it.